scripts/install.sh

```diff
@@ -1,11 +1,10 @@
-#! /bin/bash -e
+#!/bin/bash
 
 # Install helm-snyk as helm plugin getting the latest release version and copying it to
 # helm/plugins folder
-set -e
+set -eou pipefail
 
-version="$(cat plugin.yaml | grep "version" | cut -d '"' -f 2)"
-latest_version=$(curl -Is "https://github.com/snyk-labs/helm-snyk/releases/latest" | grep -i "location" | sed s#.*tag/##g | tr -d "\r")
+latest_version=$(curl -Is "https://github.com/snyk-labs/helm-snyk/releases/latest" | grep -E "^Location:" | rev | cut -d"/" -f1 | rev | tr -d "\r\n")
```

Comment:
Needs case insensitive for the header
```suggestion
latest_version=$(curl -Is "https://github.com/snyk-labs/helm-snyk/releases/latest" | grep -Ei "^Location:" | rev | cut -d"/" -f1 | rev | tr -d "\r\n")
```

Reply:
Also not sure everyone will have `rev` `cut` and `tr`. 
With sed: 
```
curl -Is "https://github.com/snyk-labs/helm-snyk/releases/latest" | grep -Ei "^Location:" | sed -E 's:^.*/::'
```

With `grep` only
```
curl -Is "https://github.com/snyk-labs/helm-snyk/releases/latest" | grep -Ei "^Location:" | grep -Eo ' \S+' | grep -Eo '[^/]+$'
```